src/config/interface.ts

```diff
@@ -0,0 +1,8 @@
+export interface IAppConfig {
+  port: number;
+  postgres_user?: string;
+  postgres_password?: string;
+  postgres_db?: string;
+  postgres_port: number;
+  postgres_host?: string;
```

Comment:
```suggestion
  postgres_user: string;
  postgres_password: string;
  postgres_db: string;
  postgres_port: number;
  postgres_host: string;
```

можливо так буде краще?